homeassistant/components/totalconnect/__init__.py

```diff
@@ -66,6 +73,16 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return unload_ok
 
 
+async def update_listener(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Update listener."""
+    if AUTO_BYPASS in entry.options:
+        client = hass.data[DOMAIN][entry.entry_id].client
+        for location_id in client.locations:
+            client.locations[location_id].auto_bypass_low_battery = entry.options[
```

Comment:
Is it going to be a problem if you don't set this value? Right now your if block would prevent these attributes from being set until AUTO_BYPASS is set in options

Reply:
good catch, thanks